package.json

```diff
@@ -54,69 +54,69 @@
   },
   "devDependencies": {
     "@resin.io/types-mochainon": "^2.0.1",
-    "@types/lodash": "4.14.121",
     "@types/mocha": "^5.2.7",
     "balena-config-karma": "^2.3.1",
-    "browserify": "^14.3.0",
+    "browserify": "^14.5.0",
     "catch-uncommitted": "^1.4.0",
     "chai-samsam": "0.0.2",
-    "coffeescript": "1.12.5",
+    "coffeescript": "~1.12.7",
     "dotenv": "^4.0.0",
-    "dtslint": "^1.0.2",
+    "dtslint": "^1.0.3",
     "gulp": "^3.9.1",
-    "gulp-coffee": "^2.3.1",
+    "gulp-coffee": "^2.3.5",
     "gulp-coffeelint": "^0.6.0",
     "gulp-insert": "^0.5.0",
-    "gulp-mocha": "^4.1.0",
+    "gulp-mocha": "^4.3.1",
     "gulp-replace": "^1.0.0",
     "gulp-tap": "^1.0.1",
     "gulp-typescript": "^5.0.1",
     "gulp-uglify": "^3.0.2",
-    "gulp-util": "^3.0.4",
-    "husky": "^3.0.0",
-    "js-polyfills": "^0.1.33",
-    "jsdoc-to-markdown": "^3.0.0",
-    "karma": "^3.0.0",
+    "gulp-util": "^3.0.8",
+    "husky": "^4.0.10",
+    "js-polyfills": "^0.1.42",
+    "jsdoc-to-markdown": "^3.0.4",
+    "karma": "^3.1.4",
     "karma-env-preprocessor": "^0.1.1",
-    "lint-staged": "^9.0.0",
-    "mocha": "^3.0.0",
+    "lint-staged": "^10.0.1",
+    "mocha": "^3.5.3",
     "mochainon": "^2.0.0",
     "prettier": "^1.19.1",
     "readable-stream": "^2.3.7",
     "require-npm4-to-publish": "^1.0.0",
     "rimraf": "^2.7.1",
     "rindle": "^1.3.4",
-    "run-sequence": "^1.1.0",
-    "superagent": "^3.0.0",
+    "run-sequence": "^1.2.2",
+    "superagent": "^3.8.3",
     "tmp": "^0.0.31",
-    "ts-node": "^3.3.0",
+    "ts-node": "^7.0.1",
     "tslint": "^5.20.1",
     "tslint-config-prettier": "^1.18.0",
-    "tslint-no-unused-expression-chai": "^0.1.3",
-    "typescript": "^3.4.5",
+    "tslint-no-unused-expression-chai": "^0.1.4",
+    "typescript": "^3.7.5",
     "uglify-es": "^3.3.9",
     "vinyl-source-buffer": "^1.1.1",
     "watch": "^1.0.2"
   },
   "dependencies": {
-    "@types/bluebird": "3.5.21",
-    "@types/common-tags": "^1.4.0",
+    "@types/bluebird": "^3.5.29",
+    "@types/common-tags": "^1.8.0",
+    "@types/lodash": "^4.14.149",
     "@types/memoizee": "^0.4.3",
     "@types/node": "^8.10.59",
     "abortcontroller-polyfill": "^1.4.0",
-    "balena-auth": "^3.0.0",
-    "balena-device-status": "^3.1.0",
-    "balena-errors": "^4.1.0",
-    "balena-hup-action-utils": "~3.0.0",
-    "balena-pine": "^9.0.0",
+    "balena-auth": "^3.0.1",
+    "balena-device-status": "^3.2.0",
+    "balena-errors": "^4.2.1",
+    "balena-hup-action-utils": "^3.0.0",
```

Comment:
The `~` here was intentional, since atm after each minor release, we wait for the proxy to pick-up the new rules.

Reply:
So a minor update is actually a breaking update?

Reply:
I've changed it back so as to not block but it still sounds problematic